calibrate_angular_node:
  ros__parameters:
    # test angle
    test_angle: 360.0
    # angular speed in radians per second
    speed: 2.0
    # error tolerance to goal angle in degrees
    tolerance: 1.5
    # angular correction factor
    odom_angular_scale_correction: 1.0
    # check to start the test
    start_test: False
  